### PR TITLE
40K 10 cards + 1 fix

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/chronomancer.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Chronomancer
+ManaCost:1 B
+Types:Artifact Creature Necron Wizard
+PT:1/1
+K:Flying
+A:AB$ Draw | PreCostDesc$ Atomic Transmutation — | Cost$ 1 T Sac<1/Artifact.Other/another artifact> | SpellDescription$ Draw a card.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/chronomancer.txt:6]: bad capping – PrecostDesc

File: forge-gui/res/cardsfolder/upcoming/chronomancer.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+K:Unearth:2 B
+DeckHas:Ability$Sacrifice|Graveyard
+DeckHints:Type$Artifact|Creature
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/chronomancer.txt:9]: why Creature?

File: forge-gui/res/cardsfolder/upcoming/necron_deathmark.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Necron Deathmark
+ManaCost:3 B B
+Types:Artifact Creature Necron
+PT:5/3
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDestroy | TriggerDescription$ Synaptic Disintegrator — When CARDNAME enters the battlefield, destroy up to one target creature and target player mills three cards.
+SVar:TrigDestroy:DB$ Destroy | ValidTgts$ Creature | TargetMin$ 0 | TargetMax$ 1 | SubAbility$ DBMill
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/necron_deathmark.txt:7]: TgtPrompt$ Select up to one target creature

File: forge-gui/res/cardsfolder/upcoming/necron_deathmark.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+SVar:DBMill:DB$ Mill | ValidTgts$ Player | TgtPrompt$ Select target player | NumCards$ 3
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/necron_deathmark.txt:8]: this one doesn't need target prompt, but it isn't hurting anything

File: forge-gui/res/cardsfolder/upcoming/necron_monolith.txt [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+PT:7/7
+K:Flying
+K:Indestructible
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ Eternity Gate — Whenever CARDNAME attacks, mill three cards. For each creature card milled this way, create a 2/2 black Necron Warrior artifact creature token.
+SVar:TrigMill:DB$ Mill | NumCards$ 3 | RememberMilled$ True | SubAbility$ DBToken
+SVar:DBToken:DB$ Token | TokenAmount$ X | TokenScript$ b_2_2_a_necron_warrior | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+K:Crew:4
+SVar:X:Remembered$Valid Creature
+DeckHas:Ability$Mill|Token
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/necron_monolith.txt:13]: `& Type$Necron|Warrior`

File: forge-gui/res/cardsfolder/upcoming/necron_overlord.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Name:Necron Overlord
+ManaCost:2 B B
+Types:Artifact Creature Necron Noble
+PT:2/5
+A:AB$ LoseLife | PreCostDesc$ Relentless Mind — | Cost$ X T tapXType<X/Artifact> | ValidTgts$ Opponent | LifeAmount$ X | SpellDescription$ Target opponent loses X life.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/necron_overlord.txt:5]: precostdesc capping

File: forge-gui/res/cardsfolder/upcoming/night_scythe.txt [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Name:Night Scythe
+ManaCost:3
+Types:Artifact Vehicle
+PT:3/1
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ Invasion Beams — When CARDNAME enters the battlefield, create a 2/2 black Necron Warrior artifact creature token.
+SVar:TrigToken:DB$ Token | TokenScript$ b_2_2_a_necron_warrior
+K:Crew:2
+Oracle:Flying\nInvasion Beams — When Night Scythe enters the battlefield, create a 2/2 black Necron Warrior artifact creature token.\nCrew 2 (Tap any number of creatures you control with total power 2 or more: This Vehicle becomes an artifact creature until end of turn.)
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/night_scythe.txt:9]: add DeckHas

File: forge-gui/res/cardsfolder/upcoming/sister_hostpitaller.txt [hunk 8]
```diff
@@ -0,0 +1,10 @@
+Name:Sister Hospitaller
+ManaCost:4 W B
+Types:Creature Human Cleric
+PT:3/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChangeZone | TriggerDescription$ Medicus Ministorum - When CARDNAME enters the battlefield, return target creature card from your graveyard to the battlefield. You gain life equal to that card's mana value.
+SVar:TrigChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Choose target creature card in your graveyard | GainControl$ True | ChangeNum$ 1 | SubAbility$ DBGainLife
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/sister_hostpitaller.txt:6]: Own better than Ctrl

Shouldn't need GainControl

File: forge-gui/res/cardsfolder/upcoming/venomcrawler.txt [hunk 9]
```diff
@@ -0,0 +1,9 @@
+Name:Venomcrawler
+ManaCost:3 B
+Types:Artifact Creature Demon
+PT:2/2
+K:Lifelink
+T:Mode$ ChangesZone | Origin$ Battlefield | ValidCard$ Creature.Other | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Devourer of Souls — Whenever another creature dies, put a +1/+1 counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1 | ConditionPresent$ Card.StrictlySelf
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/venomcrawler.txt:7]: Not sure this ConditionPresent is really needed

**Review by tool4ever — APPROVED** [review]

**Pull request merged**